src/main/java/seedu/address/model/tank/readings/AmmoniaLevel.java

```diff
@@ -12,13 +12,13 @@
 public class AmmoniaLevel extends Reading {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Ammonia Level should be a number, and it should not be blank.";
+            "Ammonia Level should be a number (max 1 d.p) between 0 and 4, and it should not be blank.";
```

Comment:
Ammonia test strips return values in multiples of .25 so this would be incorrect.

Reply:
removed 1.dp restriction